src/all-exfils.ts

```diff
@@ -137,7 +137,28 @@ const ALL_DUMPED_EXFILS_FROM_SCRIPT = {
     'Nakatani_stairs_free_exit',
   ],
 };
-
+const ALL_NEW_CUSTOM_EXFILS = {
```

Comment:
Maybe we can introduce a new config field `enable_custom_exfils` that will bypass the validation of the exfils name.

For a first iteration it seems enough.